### PR TITLE
Update docs to include example on manually validating API keys

### LINKED ISSUE #256: There is no clearly documented way to manually validate an API key
I have a web socket endpoint that various IoT devices talk to. The first web socket message must contain a valid API key otherwise the web socket connection is terminated. To support this use case, I need to manually verify if an API key is valid from within a Django Channels WebsocketConsumer.

After digging through a bunch of previous issues, PRs, and the codebase, it appears that I can use a manager method to check if a given string is a valid API key. My understanding is that this should return `True` if the `raw_key` exists, is not revoked, and is not expired; or `False` otherwise: 

```python
class MyCustomAPIKey(AbstractAPIKey):
    pass

is_valid_key = MyCustomAPIKey.objects.is_valid(raw_key)
```

Is this the correct use of that method? If so, I'll add some documentation and an example to make it more clear.

I also noticed the instance method `is_valid()` on `AbstractAPIKey` has a different implementation to `is_valid()` on `BaseAPIKeyManager` and doesn't appear to check the expiry date. Is this difference intentional?

### PR DESCRIPTION
**Pull request by jabelone**
This fixes #256

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -73,6 +73,17 @@ See also [Setting the permission policy](http://www.django-rest-framework.org/ap
     permission_classes = [HasAPIKey | IsAuthenticated]
     ```
 
+### Manually validating API keys
```
Comment (florimondmanca) [inline docs/guide.md:76]: I think we should move this to the bottom of the page, as a last section. It's a "manual mode when you need to validate yourself" so it makes sense to keep it off the "happy I-only-use-DRF-permissions" path, WDYT?

File: docs/guide.md [hunk 2]
```diff
@@ -73,6 +73,17 @@ See also [Setting the permission policy](http://www.django-rest-framework.org/ap
...
+You can also manually validate an API key with the `APIKey` objects manager. This is useful for validating API keys outside of a normal Django view, such as inside a websocket consumer from Django Channels, etc. Just use the `is_valid()` method on the manager like below.
```
Comment (florimondmanca) [inline docs/guide.md:77]: Tweaking the language a bit here. When writing docs, it's best not to use "just" or "simpli"y as things that seem obvious to us docs writers actually aren't to most readers (which is why they're documented! :) ).

```suggestion
You can also manually validate an API key with the `APIKey` objects manager using the `is_valid()` method on the manager lke below. This is useful for validating API keys outside of a normal Django view, such as inside a websocket consumer from Django Channels.
```

**Review by florimondmanca — COMMENTED** [review]
Thanks!

File: docs/guide.md (hunk 1 above)
Comment (jabelone) [reply docs/guide.md:76]: I don’t mind, I just put it where I thought I’d go looking for it. :)

**Review by florimondmanca — APPROVED** [review]
Alright, let's go with this for now. Thanks!

**Pull request merged**